hardware/raspberrypi/bootmodes/msd.md

```diff
@@ -4,7 +4,20 @@
 
 This tutorial explains how to boot your Raspberry Pi from a USB mass storage device such as a flash drive or a USB hard disk. Note that this feature does not work with all USB mass storage devices.
 
-## Program USB host boot mode (Raspberry Pi 3A+ and 3B only)
+See the [documentation](https://www.raspberrypi.org/documentation/hardware/raspberrypi/bootmodes/) for the boot sequence and alternative boot modes (Network, USB device, GPIO or SD boot).
+
+Note that "USB Mass Storage Boot" is different from "USB Device Boot Mode", the device boot mode allows a Raspberry Pi connected to a computer to boot using files on that computer.
+
+## Raspberry Pi 1, 2 (V1.1), Compute Module, Zero
+
+**These models are not supported for USB Mass Storage Boot.**
+The boot code for USB is stored in the BCM2837 device only, so the Pi 1 A/B, Pi 2 B (v1.1), and Pi Zero will all require SD cards as they are based on the BCM2835 and BCM2836. This boot code is stored in ROM (except Pi 4B) which by definition cannot be changed.
+
+An alternative is to use the 'special bootcode.bin-only boot mode' as described [here](https://www.raspberrypi.org/documentation/hardware/raspberrypi/bootmodes/). This still requires/boots from an SD-card but allows to run on an USB Device.
+
+## Raspberry Pi 2B v1.2, 3A+, 3B
+
+On the Raspberry Pi 2B v1.2, 3A+, 3B, first USB [host boot mode](https://www.raspberrypi.org/documentation/hardware/raspberrypi/bootmodes/host.md) should be enabled. This is to allow Mass Storage Boot / Network boot (Network boot not supported on 3A+).
```

Comment:
And also the CM3 ?

Reply:
Again, probably not needed, since the usual method of getting to this page is via the boot modes page.